packages/consensus/src/types/transaction.cairo

```diff
@@ -127,6 +127,25 @@ pub struct TxOut {
     pub cached: bool,
 }
 
+///
+/// Custom implementation of the Hash trait for TxOut removed cached field.
+///
+impl TxOutHash<S, impl SHashState: HashStateTrait<S>, +Drop<S>> of Hash<TxOut, S, SHashState> {
+    #[inline(always)]
+    fn update_state(state: S, value: TxOut) -> S {
+        let state = Hash::update_state(state, value.value);
```

Comment:
I think you can do `let state = state.update(value.value)`

Reply:
but I can't do the same for @ByteArray. Looks like I need to add trait implementation in the core::hash library.

Reply:
I mean I can. But will require to create additional PR. Let me know if that is a requirement.

Reply:
Np, let's keep as is